Modules/Core/Common/include/itkMacro.h

```diff
@@ -427,7 +427,7 @@ destructor." (Visual Studio 2022/MSVC)
 #define itkOverrideGetNameOfClassMacro(thisClass) \
   const char * GetNameOfClass() const override itkInternalGetNameOfClassImplementationMacro(thisClass)
 
-#ifdef ITK_FUTURE_LEGACY_REMOVE
+#ifdef ITK_LEGACY_REMOVE
```

Comment:
Perhaps skip this change, see https://github.com/InsightSoftwareConsortium/ITK/pull/4373#issuecomment-1864190603.